synchronization/src/cyface/java/de/cyface/synchronization/Constants.java

```diff
@@ -18,6 +18,23 @@
     // This may be used by all implementing apps, thus, public
     public final static String AUTH_TOKEN_TYPE = "de.cyface.auth_token_type";
 
+    // FIXME: Duplicate constants - move ErrorHandler to SDK
```

Comment:
If everything else runs fine I would propose to do this before we merge this branch into dev.

Reply:
done